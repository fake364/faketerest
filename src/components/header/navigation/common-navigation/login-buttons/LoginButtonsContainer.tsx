import React from 'react';
import LoginButtons from './LoginButtons';

type Props = { className?: string };

const LoginButtonsContainer: React.FC<Props> = ({ className, children }) => {
	return (
		<div className={className}>
			{children}
			<LoginButtons />
		</div>
	);
};

export default LoginButtonsContainer;
