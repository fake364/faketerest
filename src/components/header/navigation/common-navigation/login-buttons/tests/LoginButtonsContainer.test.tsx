import React from 'react';

import { shallow } from 'enzyme';
import LoginButtonsContainer from '../LoginButtonsContainer';

describe('LoginButtonsContainer', () => {
  it('should render something in container and login buttons', () => {
    const wrapper = shallow(<LoginButtonsContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});
