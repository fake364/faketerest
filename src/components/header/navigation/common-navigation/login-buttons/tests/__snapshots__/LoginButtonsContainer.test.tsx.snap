// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginButtonsContainer should render something in container and login buttons 1`] = `
<div>
  <LoginButtons />
</div>
`;
